src/content/en/2021/third-parties.md

```diff
@@ -1,20 +1,491 @@
+
+Hopefully highlighting some of these well-known examples will cause readers to investigate the impact of third-party embeds on their own sites and ask themselves if they really are all worth it? Perhaps if we make this subject more important to the third parties, they will prioritize performance.
+
+### Timing-Allow-Origin header prevalence
+
+Last year we looked at the [prevalence of the `timing-allow-origin` header](./third-parties#timing-allow-origin-prevalence), which allows the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Resource_Timing_API/Using_the_Resource_Timing_API">Resource Timing API</a> to be used on third-party requests. Without this HTTP header, the information available to on-page performance monitoring tools for third-party requests is restricted for security and privacy reasons. However, for static requests, third parties which allow this header give a greater transparency into the performance timings of their resources.
+
+{{ figure_markup(
+  image="third-parties-timing-allow-origin-header-usage.png",
+  caption="Timing-Allow-Origin header usage.",
+  description="Bar chart showing the prevalence of the `timing-allow-origin` HTTP header on third-party requests over the last three years for desktop and mobile. In July 2019 this header was on 30.0% of desktop requests, and 31.9% of mobile requests. In August 2020 this increased to 31.5% and 31.3% respectively, but in July 2021 this decreased to 27.8% of desktop third-party requests and 26.7% of mobile third-party requests.",
+chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkT-CF5-NB7Oatd6XZq_08EoMGfiygKZtW4OwVivaaW3cIlt3ZcWNBtOdNePD_nKzkvb2nMlhWOX6g/pubchart?oid=626106432&format=interactive",
+  sheets_gid="1788549043",
+  sql_file="tao_by_third_party.sql"
+  )
+}}
+
+Looking at the usage over the last three Web Almanac years usage has dropped considerably this year, instead of increasing, which is disappointing from a transparency and analysis perspective. Digging deeper into the data showed a 33% drop in Facebook requests and, as they supported this header and are widely used, this explains most of this drop. Interestingly the number of pages with Facebook usage actually increased, but it looks like Facebook have changed their embed to make fewer requests in the last year and, given their prevalence, that's made quite a dent on the usage of timing-allow-origin header. Ignoring that, usage of this header has basically stayed stable—which is a bit disappointing given the focus on performance with <a hreflang="en" href="https://developers.google.com/search/blog/2020/11/timing-for-page-experience">the ranking impact of the Core Web Vitals</a>.
+
+## Security and Privacy
```

Comment:
How can we make this section more data-driven?

Reply:
I spent a good bit of time on this, honestly I did, trying to come up with something more data driven, but unfortunately did not. It's apparently the last two year's chapters had the same problem.

None-the-less, I think they are important topics to cover, and think the chapter would be incomplete without this section.